src/s3/listobjects.cpp

```diff
@@ -32,18 +35,19 @@ asio::awaitable<void> irods::s3::actions::handle_listobjects_v2(
 {
     using namespace boost::property_tree;
 
-    auto thing = irods::s3::get_connection();
+    auto rcComm_t_ptr = irods::s3::get_connection();
```

Comment:
If this is a pointer, should we use `auto*` here?

Reply:
Hmm, is that really a pointer?

Reply:
Not sure. It says "_ptr" in the name and it is being dereferenced with an asterisk later, so it checked several boxes :)

Reply:
It is a smart pointer (std::unique_ptr).  Will auto* work with smart pointers?

Reply:
Just verified that auto* will not work with the smart pointer.